### PR TITLE
Fix flags values

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/468

Don't builds until gtk, gdk, ~sys~ regened with https://github.com/gtk-rs/gir/pull/471

cc @GuillaumeGomez , @sdroege

EDIT: sys don't need updated for this PR works.

### CODE REVIEW

**Comment by sdroege** [comment]
Looks good to me, and much nicer API-wise

**Comment by EPashkin** [comment]
@GuillaumeGomez added minor update to Cargo.toml to simplify local build test.
Build will fails so don't need wait for CI.

File: Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ git = "https://github.com/gtk-rs/gdk"
 git = "https://github.com/gtk-rs/gtk"
 
 [features]
-#default = ["gtk_3_20"]
+#default = ["gtk_3_22"]
```
Comment (GuillaumeGomez) [inline Cargo.toml]: Ah, new default?

**Comment by GuillaumeGomez** [comment]
Yes sir!

**Pull request merged**